src/components/box/BBoxItem.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Typography } from "@mui/material";
+import _ from "lodash";
+
+interface BBoxItemProps {
+  label: string;
+  value: number;
+}
+
+const BBoxItem: React.FC<BBoxItemProps> = ({ label, value }) => {
+  const roundedValue = _.round(value, 2);
+  return (
+    <Typography
+      sx={{
+        color: "#7194AB",
```

Comment:
Use color constant or styled component?

Reply:
Because there is already a new design update : https://github.com/aodn/backlog/issues/5991#issuecomment-2516246028 and i was told the design will also be discussed and might change further, in addition the #7194AB is a new color, so i hard coded it and will change according to new design in the future.